src/containers/TaskRun/TaskRun.js

```diff
@@ -38,14 +38,16 @@ import {
   updateUnexecutedSteps
 } from '@tektoncd/dashboard-utils';
 
-import { fetchLogs, getViewChangeHandler } from '../../utils';
+import { fetchLogs, followLogs, getViewChangeHandler } from '../../utils';
+
 import { LogDownloadButton } from '..';
 import {
   getSelectedNamespace,
   getTaskByType,
   getTaskRun,
   getTaskRunsErrorMessage,
-  isWebSocketConnected
+  isWebSocketConnected,
+  streamLogs
```

Comment:
can we rename this to make it clear it's a boolean? `isLogStreamingEnabled` or something like that

Reply:
Fixed, thanks